IDE/ECLIPSE/DEOS/user_settings.h

```diff
@@ -103,8 +105,6 @@ You can get the current time from https://www.unixtimestamp.com/
 
 #endif
 
-#define printf printx
```

Comment:
IIRC, printx was only available for development. I see that you've moved it to the benchmark and test code which is ok but be sure not to include it for the default crypto/ssl build.

Reply:
I confirmed that removing the `printx` dependency and building without test and benchmark still works with DTLS 1.2 that I've also been testing.

```
#define NO_CRYPT_TEST
#define NO_CRYPT_BENCHMARK
```